packages/mjml-social/README.md

```diff
@@ -93,7 +93,6 @@ text-padding                | px          | padding around the text       | 4px
 sizes                       | media query & width | set icon width based on query | n/a
 src                         | url         | image source                  | Each social `name` has its own default
 srcset                      | url & width | set a different image source based on the viewport | n/a
-rel                         | string      | specify the rel attribute for the link    | n/a
```

Comment:
Looks to be used in SocialElement code too, can you add it into allowedAttributes too ?